src/decision_model.jl

```diff
@@ -34,7 +34,7 @@ function is_forbidden(s::Path, forbidden_paths::Vector{ForbiddenPath})
     return !all(s[k]∉v for (k, v) in forbidden_paths)
 end
 
-function path_probability_variable(model::Model, z::DecisionVariables, s::Path, P::AbstractPathProbability, hard_lower_bound::Bool, forbidden::Bool, base_name::String="")
+function path_probability_variable(model::Model, z::DecisionVariables, s::Path, P::AbstractPathProbability, hard_lower_bound::Bool, forbidden::Bool, probability_scale_factor::Float64, base_name::String="")
```

Comment:
Should this also have a default value specified?

Reply:
I didn't add a default value here because path_probability_variable is a helper function called by the PathProbabilityVariables function. I set the default value in the PathProbabilityVariables function. I decided to it this way because now it's consistent with the other parameters hard_lower_bound and forbidden. The base_name has been done differently, I don't know why. Do you think the default values should be added to this function as well?

Reply:
Makes sense, it's probably good that it is consistent with the other parameters.